companies/tradedoubler.json

```
@@ -0,0 +1,40 @@
+{
+    "slug": "tradedoubler",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Tradedoubler AB",
+    "runs": [
+        "Tradedoubler GmbH",
+        "Adnologies",
+        "Interactive Marketing Works Ltd"
+    ],
+    "address": "Birger Jarlsgatan 57A\n113 56 - Stockholm\nSweden\n",
```

```suggestion
    "address": "Birger Jarlsgatan 57A\n113 56 Stockholm\nSweden\n",
```

All previous records with Swedish addresses are without the dash.